src/main/resources/anniv5Resources/localization/eng/madsciencepack/Cardstrings.json

```diff
@@ -29,7 +29,7 @@
   },
   "${ModID}:Cheapen": {
     "NAME": "Cheapen",
-    "DESCRIPTION": "${ModID}:Modify a random card in your draw pile: 'Costs 0.' NL Exhaust, unless Amplify 2."
+    "DESCRIPTION": "${ModID}:Modify a random card in your draw pile: 'Costs 0.' NL Exhaust, (NYI) unless Amplify 2."
```

Comment:
I've already changed cheapen in my Marisa pack, please remove your change here or it'll lead to merge conflicts